engines/cpu.c

```diff
@@ -8,11 +8,26 @@
 #include "../fio.h"
 #include "../optgroup.h"
 
+// number of 32 bit integers to sort
+size_t qsort_size = (256 * (1ULL << 10)); // 256KB
+
+typedef struct {
```

Comment:
Lets get rid of this typedef, generally not a fan unless it's a specific single unit type.

Reply:
sure, done.